docs/docsite/rst/user_guide/intro_getting_started.rst

```diff
@@ -64,8 +64,13 @@ do:
 
     $ ssh-agent bash
     $ ssh-add ~/.ssh/id_rsa
+    
+Depending on your setup, you may wish to use Ansible's ``--private-key`` command line option to specify a pem file instead.  You can also add the private key file:
 
-(Depending on your setup, you may wish to use Ansible's ``--private-key`` option to specify a pem file instead)
+    $ ssh-agent bash
+    $ ssh-add ~/.ssh/keypair.pem
+
+Another way to add private key files without using ssh-agent is using ``ansible_ssh_private_key_file`` in an inventory file as explained here:  :doc:`intro_inventory`.
```

Comment:
Thanks @thadguidry for the PR. It's always wonderful to get edits from users who are learning Ansible, or helping others learn Ansible.

We'd like to use `:ref:` links (which update automatically if the content moves from one file to another) rather than `:doc:` links (which point to a specific path and filename) in the docs. See https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#internal-navigation for more details. 
```suggestion
Another way to add private key files without using ssh-agent is using ``ansible_ssh_private_key_file`` in an inventory file as explained here:  :ref:`intro_inventory`.
```

Reply:
Thanks. DONE!  I hope the reading flow is not broken before the "now ping all your nodes:".  1 person suggested that numbering steps or some such might be useful, but I didn't want to break the general format too much from original.